services/blockchain-app-registry/shared/utils/fsUtils.js

```diff
@@ -179,6 +179,25 @@ const rename = async (oldName, newName) => new Promise((resolve, reject) => {
 	});
 });
 
+const deleteEmptyFoldersAndNonMetaFiles = (directoryPath) => {
```

Comment:
Could we please this method asynchronous. Please use the async versions for the following methods: `readdirSync`, `statSync`, `readdirSync` and `unlinkSync`. You can refer to the existing implementations for the same in other microservices. If they do not exist, please implement the same in `fsUtils.js`. Please make sure to add tests for any methods you add to fsUtils.